__tests__/app.test.js

```diff
@@ -69,16 +69,76 @@ describe("GET /api/reviews", () => {
         });
       });
   });
-  test("200: reviews are sorted by newest first", () => {
+  test("200: reviews are sorted by newest first by default", () => {
     return request(app)
       .get("/api/reviews")
+      .expect(200)
       .then(({ body: { reviews } }) => {
-        expect(reviews).toBeSortedBy(reviews.created_at, {
+        expect(reviews).toBeSortedBy("created_at", {
           descending: true,
           coerce: true,
         });
       });
   });
+  describe("Queries", () => {
+    describe("Category", () => {
+      test("200: Filter by specified category", () => {
+        return request(app)
+          .get("/api/reviews?category=social deduction")
+          .expect(200)
+          .then(({ body: { reviews } }) => {
+            expect(reviews).toHaveLength(11);
+            reviews.forEach((review) => {
+              expect(review.category).toBe("social deduction");
+            });
+          });
+      });
+      test("200: does not allow SQL injection", () => {
+        return request(app)
+          .get("/api/reviews?category=euro game ;")
+          .expect(200)
+          .then(({ body: { reviews } }) => {
+            expect(reviews).toEqual([]);
+          });
+      });
+    });
+    describe("Sort_by", () => {
+      test("200: retrieves all reviews in order of specified column", () => {
+        return request(app)
+          .get("/api/reviews?sort_by=votes")
+          .expect(200)
+          .then(({ body: { reviews } }) => {
+            expect(reviews).toBeSorted({ key: "votes", descending: true });
+          });
+      });
+      test("200: does not allow SQL injection", () => {
+        return request(app).get("/api/reviews?sort_by=votes ;").expect(200);
+      });
+    });
+    describe("order", () => {
+      test("200: can set order to ascend", () => {
+        return request(app)
+          .get("/api/reviews?order=asc")
+          .expect(200)
+          .then(({ body: { reviews } }) => {
+            expect(reviews).toBeSorted({ key: "created_at" });
+          });
+      });
+      test("200: does not allow SQL injection", () => {
+        return request(app).get("/api/reviews?order=asc ;").expect(200);
+      });
+    });
+    test("200: all queries work in conjunction", () => {
+      return request(app)
+        .get("/api/reviews?category=social deduction&sort_by=votes&order=asc")
+        .expect(200)
+        .then(({ body: { reviews } }) => {
+          expect(reviews.length).toBe(11);
+
+          expect(reviews).toBeSorted({ key: "votes" });
+        });
+    });
+  });
```

Comment:
Impressive demonstration of the happy path working great! Well done 😀 

You are missing one happy path test, what happens if you filter by a category that has no reviews? 

I would now add sad path testing: 

What happens if I sort by colour? 
What happens if i order by horizontally?